v1.1.yml

```diff
@@ -1517,6 +1659,13 @@ definitions:
         format: datetime
         x-nullable: true
 
+  Credentials:
+    type: object
+    properties:
+      district_username:
+        type: string
+        x-nullable: true
+
```

Comment:
You've handled `credentials.district_username` and `google_apps.email` differently here.

I think the way we do it with GoogleApps is correct. Either the whole object is there or not. Meaning this `x-nullable: true` should be removed